master/getting-started/openshift/installation.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Installing Calico on OpenShift
+---
+
+Installation of Calico in OpenShift is integrated into the openshift-ansible scripts.
+
+## Basic
+
+For a basic Calico installation in OpenShift that shares the etcd
+instance used by the apiserver, set the following two `OSev3:vars` in your
+inventory file:
+
+  - `os_sdn_network_plugin_name=cni`
+  - `openshift_use_calico=true`
+
```

Comment:
The variable `openshift_use_openshift_sdn=false ` has to be set otherwise it fail due to `  when: openshift_use_openshift_sdn | default(true) | bool and openshift_use_calico | default(false) | bool` in `roles/openshift_common/tasks/main.yml`.